src/Tests/Topology/When_configuring_publishers.cs

```diff
@@ -34,7 +34,7 @@ public void Should_not_map_publisher_and_type_if_it_is_not_a_message()
 
             var exception = Assert.Throws<InvalidOperationException>(() => configuration.GetPublishersFor(typeof(MyType)));
             StringAssert.Contains($"No publishers configured for `{typeof(MyType).FullName}`", exception.Message);
-            StringAssert.Contains($"Please add mappings via `endpointConfiguration.UseTransport<AzureServiceBusTransport>().UseTopology<EndpointOrientedTopology>().RegisterPublisherForType(\"endpointname\", typeof({typeof(MyType).FullName}))`", exception.Message);
+            StringAssert.Contains($"Register mappings via `endpointConfiguration.UseTransport<AzureServiceBusTransport>().UseTopology<EndpointOrientedTopology>().RegisterPublisherForType(\"endpointname\", typeof({typeof(MyType).FullName}))`", exception.Message);
```

Comment:
I think I get it now. The test is to verify that a non-message type won't get registered, and the way to test it is to generate an exception. Then just removing the message assertion is good.